src/server/auth/server/api_server.go

```diff
@@ -258,7 +264,13 @@ func (a *apiServer) getClusterRoleBinding(ctx context.Context) (*auth.RoleBindin
 }
 
 func (a *apiServer) isActive(ctx context.Context) error {
-	_, err := a.getClusterRoleBinding(ctx)
+	return a.txnEnv.WithReadContext(ctx, func(txnCtx *txnenv.TransactionContext) error {
+		return a.isActiveInTransaction(txnCtx)
```

Comment:
Is it free to open a transaction if we don't actually use it? The happy path for `getClusterRoleBinding` is that it hits the in-memory cache, so I just want to make sure this isn't adding overhead in that case.

Reply:
not free :( 

thanks for calling this out